internal/pkg/docker/dockercompose/decompose.go

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/loader"
+	compose "github.com/compose-spec/compose-go/types"
+	"sort"
+)
+
+// DecomposeService parses a Compose YAML file and then converts a single service to a Copilot manifest.
+func DecomposeService(content []byte, svcName string, workingDir string) (*manifest.LoadBalancedWebService, *manifest.BackendService, IgnoredKeys, error) {
```

Comment:
4 return values is a lot - maybe combine the two manifest types into a struct to return instead?